README.md

```diff
@@ -21,15 +21,10 @@ files.
 
 | Signal  | Status     |
 | ------- | ---------- |
-| Logs    | Stable*    |
+| Logs    | Stable     |
```

Comment:
This is great! Can we replace the table with a simple statement "stable across logs/metrics/traces"?

Reply:
updated: is this better?